### PR TITLE
feat: get fixed fonts from s3

### PR DESCRIPTION
**Pull request by L0wry**
we cant fix fonts on bitrise because font forge is not available on linux.

As a result, this PR will allow bitrise to get a set of fonts that have already been fixed from s3.

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/23111b2f04b9998e3b0a3a38c7c7c41ef82cfd8d/index.html

File: lib/fetch-fixed-fonts.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const fs = require("fs");
+const AWS = require("aws-sdk");
+const { promisify } = require("util");
+const mkdirp = promisify(require("mkdirp"));
+
+const s3 = new AWS.S3();
+AWS.config.update({ region: "eu-west-1" });
+
+const fontDir = `${__dirname}/../dist/public/fonts`;
```
Comment (craigbilner) [inline lib/fetch-fixed-fonts.js]: would typically use `path.resolve`

File: lib/fetch-fixed-fonts.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+const bucket = "nu-tools-fonts";
+
+s3.listObjects({ Bucket: bucket }, (error, bucketObjects) => {
+  if (error) throw error;
+
+  mkdirp(fontDir);
```
Comment (craigbilner) [inline lib/fetch-fixed-fonts.js]: this is async?

File: lib/fetch-fixed-fonts.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+  const contents = bucketObjects.Contents;
```
Comment (craigbilner) [inline lib/fetch-fixed-fonts.js]: can destructure?

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/5ccebea765cfbc98aaf09de41cb20ee9f378610d/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/b363bbb0551584b4bb5ce5b0192ab71f7275650e/index.html

**Pull request closed without merging**